datalad/interface/tests/test_rerun.py

```diff
@@ -757,6 +758,158 @@ def test_run_inputs_no_annex_repo(path):
     ds.rerun()
 
 
+@slow  # ~10s
+@known_failure_windows
+@with_testrepos('basic_annex', flavors=['clone'])
+def test_run_explicit(path):
```

Comment:
This is wrong for two reasons: you introduce test_run_explicit *twice* in merge, and test_run_explicit is already included in core/local/tests/test_run.py.

Reply:
Oops, sorry. May be you could just redo this PR? I will not get to the laptop for a bit

Reply:
I've updated your branch.